### PR TITLE
Update TypeLoadException text for Type.GetType

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to dotnet/docs#39594.

### CODE REVIEW

File: xml/System/Type.xml [hunk 1]
```diff
@@ -8970,7 +8971,7 @@ The following example retrieves the `Type` object of a user-defined class, retri
...
+> Processor architecture is added to assembly identity, and can be specified as part of assembly name strings. For example, "ProcessorArchitecture=msil". However, it is not included in the string returned by the <xref:System.Type.AssemblyQualifiedName%2A> property, for compatibility reasons. You can also load types by creating an <xref:System.Reflection.AssemblyName> object and passing it to an appropriate overload of the <xref:System.Reflection.Assembly.Load%2A> method. You can then use the <xref:System.Reflection.Assembly.GetType%2A?displayProperty=nameWithType> method to load types from the assembly. See also <xref:System.Reflection.AssemblyName.ProcessorArchitecture%2A?displayProperty=nameWithType>.
```
Comment (jkotas) [inline xml/System/Type.xml:8974]:
```suggestion
> Processor architecture is part of assembly identity, and can be specified as part of assembly name strings. For example, "ProcessorArchitecture=msil". However, it is not included in the string returned by the <xref:System.Type.AssemblyQualifiedName%2A> property, for compatibility reasons. You can also load types by creating an <xref:System.Reflection.AssemblyName> object and passing it to an appropriate overload of the <xref:System.Reflection.Assembly.Load%2A> method. You can then use the <xref:System.Reflection.Assembly.GetType%2A?displayProperty=nameWithType> method to load types from the assembly. See also <xref:System.Reflection.AssemblyName.ProcessorArchitecture%2A?displayProperty=nameWithType>.
```

File: xml/System/Type.xml [hunk 2]
```diff
@@ -9069,7 +9068,7 @@ Type.GetType("System.Collections.Generic.Dictionary`2[System.String,[MyType,MyAs
 
  <paramref name="typeName" /> represents a generic type, and one of its type arguments does not satisfy the constraints for the corresponding type parameter.</exception>
         <exception cref="T:System.TypeLoadException">
-          <paramref name="typeName" /> represents an array of <see cref="T:System.TypedReference" />.</exception>
+          <paramref name="typeName" /> represents an array of <see cref="T:System.TypedReference" /> or an invalid type.</exception>
```
Comment (jkotas) [inline xml/System/Type.xml:9071]:
```suggestion
          <paramref name="typeName" /> represents an invalid type, for example an array of <see cref="T:System.TypedReference" />.</exception>
```

File: xml/System/Type.xml [hunk 3]
```diff
@@ -9649,7 +9648,7 @@ Type.GetType("System.Collections.Generic.Dictionary`2[System.String,[MyType,MyAs
...
```
Comment (jkotas) [inline xml/System/Type.xml:9651]:
```suggestion
          <paramref name="typeName" /> represents an invalid type, for example an array of <see cref="T:System.TypedReference" />.</exception>
```

**Review by BillWagner — APPROVED** [review]
This LGTM, once you address @jkotas 's feedback.

**Pull request merged**